Modules/ConfigOptions.lua

```diff
@@ -797,6 +797,12 @@ return {
 		modList:NewMod("Multiplier:CorpseConsumedRecently", "BASE", val, "Config", { type = "Condition", var = "Combat" })
 		modList:NewMod("Condition:ConsumedCorpseRecently", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
+	{ var = "multiplierWarcryCastRecently", type = "count", label = "# of Warcies Cast Recently:", ifMult = "WarcryCastRecently", implyCondList = {"UsedWarcryRecently", "UsedWarcryInPast8Seconds", "UsedSkillRecently"}, tooltip = "This also implies you have 'Used a Warcry Recently', 'Used a Warcry in the past 8 seconds', and 'Used a Skill Recently'", apply = function(val, modList, enemyModList)
```

Comment:
Semantic correction: Warcries are Used, not Cast. It was changed in 3.11.1 so that the tooltip says "Use time" instead of "Cast time".